hathor/transaction/storage/rocksdb_storage.py

```diff
@@ -83,6 +84,9 @@ def _load_from_bytes(self, tx_data: bytes, meta_data: bytes) -> 'BaseTransaction
         tx.storage = self
         return tx
 
+    def _build_indexes_manager(self) -> IndexesManager:
```

Comment:
Yes. I understand the limitation but I think it makes sense to do this for now. It might be simple for now to support mixing both, but it won't be trivial in the long run (we will need to be able to optionally rebuild indexes). I think we could revise this feature later.